stable/artifactory/README.md

```diff
@@ -286,6 +286,28 @@ So in order to connect to Artifactory with jconsole you should map the Artifacto
 jconsole artifactory-<release-name>:<jmx-port>
 ```
 
+### Access creds. bootstraping
+**IMPORTANT:** Bootsrapping access creds. would require Artifactory pod restart which by default will generate a random DB password if the password was not provided on initial setup.
```

Comment:
@nimerb This is not relevant, each upgrade will generate a new db password and I don't think it makes sense to write it down in every upgrade use case. Please remove